src/common/platform/kernel_mapped.hpp

```diff
@@ -863,39 +863,37 @@ typedef struct _SYSTEM_TIMEOFDAY_INFORMATION64
     ULONGLONG SleepTimeBias;
 } SYSTEM_TIMEOFDAY_INFORMATION64, *PSYSTEM_TIMEOFDAY_INFORMATION64;
 
-#ifndef OS_WINDOWS
-typedef struct _SYSTEMTIME
+typedef struct _SYSTEMTIME64
 {
     WORD wYear;
     WORD wMonth;
-    WORD wDayOfWeek;
     WORD wDay;
     WORD wHour;
     WORD wMinute;
     WORD wSecond;
     WORD wMilliseconds;
-} SYSTEMTIME, *PSYSTEMTIME, *LPSYSTEMTIME;
-#endif
+    WORD wDayOfWeek;
```

Comment:
oof, interesting. would you mind adding a test for that? the unit tests use a fake clock, so it should be easy to set that to a fixed value